src/k8s-engine/lima.ts

```diff
@@ -575,7 +571,7 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     return this.status.then(defined);
   }
 
-  private calcRandomTag(desiredLength: number) {
+  static calcRandomTag(desiredLength: number) {
     // quicker to use Math.random() than pull in all the dependencies utils/string:randomStr wants
     return Math.random().toString().substr(0, desiredLength);
```

Comment:
This is incorrect (`Math.random()` returns a float, so it could be _shorter_ than `desiredLength`).

Reply:
That's ok. And I want the tag to start after the decimal point. In the case where the number has fewer than `desiredLength + 2` digits, the substr will return whatever it can. In that extremely rare case where it returns `0`, we'll get an empty string tag, which is also ok.